components/FormCreate.js

```diff
@@ -88,10 +89,11 @@ export default function FormCreate() {
     event.preventDefault();
     setSearchTerm(event.target.value);
     setIncrement(0);
+    window.scroll({ top: 500, left: 0, behavior: 'smooth' });
   };
 
   const API = process.env.NEXT_PUBLIC_IMAGE_API_KEY;
-  const { data: imageSearch } = useSWR(
+  const { data: imageSearch, isLoading } = useSWR(
     `https://pixabay.com/api/?key=${API}&q=${searchTerm}&image_type=photo`
   );
```

Comment:
Here you are exposing your pixabay API Key to the public. Please use the save variant instead: create a dedicated API endpoint in your next app (pages/api/images/[searchTerm].js for example). Then do a fetch request to pixabay from inside this endpoint. Please refer to the PR from Dominik for more information.